alacritty/src/renderer/builtin_font.rs

```diff
@@ -20,12 +20,20 @@ pub fn builtin_glyph(
     character: char,
     metrics: &Metrics,
     offset: &Delta<i8>,
+    glyph_offset: &Delta<i8>,
 ) -> Option<RasterizedGlyph> {
-    match character {
+    let mut glyph = match character {
         // Box drawing characters and block elements.
-        '\u{2500}'..='\u{259f}' => Some(box_drawing(character, metrics, offset)),
-        _ => None,
-    }
+        '\u{2500}'..='\u{259f}' => box_drawing(character, metrics, offset),
+        _ => return None,
+    };
+
+    // Since we want to ignore `glyph_offset` for builtin font apply it to compensate its addition
+    // when loading glyphs in the renderer.
```

Comment:
```suggestion
    // Since we want to ignore `glyph_offset` for the built-in font, subtract it to compensate its addition when loading glyphs in the renderer.
```